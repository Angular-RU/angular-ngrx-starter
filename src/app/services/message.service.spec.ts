import { TestBed, inject } from '@angular/core/testing';

import { MessageService } from './message.service';

describe('MessageService', () => {
  let messageService: MessageService;

  beforeEach(() => {
    messageService = new MessageService();
  });

  it('should add message', () => {
    messageService.add('test message');

    expect(messageService.messages).toMatchSnapshot();
  });

  it('should clear messages', () => {
    messageService.add('test message');
    messageService.clear();
    
    expect(messageService.messages).toMatchSnapshot();
  });
});
