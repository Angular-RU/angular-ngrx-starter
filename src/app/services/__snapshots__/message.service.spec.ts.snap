// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageService should add message 1`] = `
Array [
  "test message",
]
`;

exports[`MessageService should clear messages 1`] = `Array []`;
